src/request-handler/lib/collapse_include.js

```diff
@@ -1,4 +1,5 @@
-export default function (includes) {
+export default function (input) {
+  const includes = Array.isArray(input) ? input : input.split(',');
```

Comment:
This doesn't account for if the value is an object, which could happen with `?include[thing]=x&include[otherthing]=y`